### DISCUSSION TITLE
Can't install latest Avalonia with .NET Framework 4.6.1

### DISCUSSION THREAD

**Discussion by softempire** [AvaloniaUI/Avalonia#7851] [General]
Because Avalonia.Remote.Protocol can't be installed with .NET Framwork 4.6.1 and Avalonia depends on that

**Comment by rstm-sf** [comment]
Hello, could you provide an example?

**Comment by kekekeks** [comment]
`Avalonia.Remote.Protocol` targets `netstandard2.0` which is compatible with `net461`. Make sure that you use the latest SDK and tooling.

**Comment by softempire** [comment]
It works now after I use VS2019. Thank you!
